src/components/Header/Header.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFire } from '@fortawesome/free-solid-svg-icons';
 
```

Comment:
Cool. I didn't know you could import the font awesome package and then use the icons as components themselves. Thanks for showing me this.